awx/ui/test/e2e/tests/test-form-error-handling.js

```diff
@@ -0,0 +1,202 @@
+/* Tests for validation checks during form creation and editing. */
+
+import {
+    getJobTemplate
+} from '../fixtures';
+
+let data;
+
+const templatesNavTab = '//at-side-nav-item[contains(@name, "TEMPLATES")]';
+const orgsNavTab = '//at-side-nav-item[contains(@name, "ORGANIZATIONS")]';
+
+module.exports = {
+    before: (client, done) => {
+        const resources = [
+            getJobTemplate('test-form-error-handling'),
+            getJobTemplate('test-form-error-handling-2')
+        ];
+
+        Promise.all(resources)
+            .then(([jt, jt2]) => {
+                data = { jt, jt2 };
+                done();
+            });
+        client
+            .login()
+            .waitForAngular()
+            .resizeWindow(1200, 1000)
+            .useXpath()
+            .findThenClick(templatesNavTab)
```

Comment:
As a general rule, I prefer to navigate by URL to the page when possible. Given how little coverage we have, this is great at catching any general regressions within the implicit navigation action here.